packit/config/package_config.py

```diff
@@ -351,9 +355,34 @@ def parse_loaded_config(
             raw_dict=loaded_config,
             config_file_path=config_file_path,
             repo_name=repo_name,
+            spec_file_path=spec_file_path,
             validate=True,
         )
         return package_config
     except Exception as ex:
         logger.error(f"Cannot parse package config. {ex}.")
         raise Exception(f"Cannot parse package config: {ex}.")
+
+
+def get_local_specfile_path(directories):
```

Comment:
could you please make it more clear what directories is?

Reply:
Not sure if Tomas' comment should have already been fixed, but I think he meant to add a type hint (`List[str]`?)